### DISCUSSION TITLE
Looking for Collaborators

### DISCUSSION THREAD

**Discussion by windwp** [nvim-pack/nvim-spectre#126] [Announcements]
Hi i am looking for someone can join nvim-pack organizations.
i am kinda to busy in future and maybe someone interesting to improve and help maintain this plugin.
nvim-spectre is stabilize it doesn't need to work to much. I still think it lack some feature so if someone interesting .
you can contact me. 

Thank

**Comment by EpsilonKu** [comment]
Uhh, will try(this holiday or next) to PR about highlight. I have xp well only on colorscheme. But made some PR's on other plugins.

**Comment by EpsilonKu** [comment]
@windwp hi, can you disable trace bot? Also can I add plugins to this group?

**Comment by windwp** [comment]
what do you mean about trace bot? yes you can add  plugins to this group.
